### DISCUSSION TITLE
How to install this?

### DISCUSSION THREAD

**Discussion by Admin-n-d** [escardel/shortercut#2] [General]
Im a in IT for like 30 years, and as far as I see I need to learn VB only to launch this 'program'? There is no setup.exe? Why?

**Comment by escardel** [comment]
Click on "Releases" on the right sidebar for a .exe or on the README header link for the website with setup file.  I'll add a direct link to the README so it's easier.


________________________________
From: Michał Wojciul ***@***.***>
Sent: Saturday, March 16, 2024 12:24:48 PM
To: escardel/shortercut ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [escardel/shortercut] How to install this? (Discussion #2)


Im a in IT for like 30 years, and as far as I see I need to learn VB only to launch this 'program'? There is no setup.exe? Why?
